src/clj/rems/api/schema.clj

```diff
@@ -6,21 +6,21 @@
 
 (s/defschema CatalogueItem
   {:id s/Int
-   :title s/Str
    :wfid s/Int
    (s/optional-key :workflow-name) s/Str
    :formid s/Int
    (s/optional-key :form-name) s/Str
    :resid s/Str
    :resource-id s/Int
    (s/optional-key :resource-name) s/Str
-   (s/optional-key :langcode) s/Keyword
    :start DateTime
    :end (s/maybe DateTime)
    :enabled s/Bool
    :archived s/Bool
    :expired s/Bool
-   :localizations (s/maybe {s/Any s/Any})})
+   :localizations (s/maybe {s/Keyword {:id s/Int
+                                       :langcode s/Keyword
+                                       :title s/Str}})})
```

Comment:
Could `s/maybe` be removed?

Reply:
I considered that, but the problem is that API allows adding a catalogue item without adding localizations, which would then cause an error when fetching the catalogue item. Maybe the API could be changed so that localizations would be added with a single API call, but that would probably be another patch...

Reply:
An empty map should pass the schema without s/maybe

Reply:
Ah, true. I'll do it that way.